stable/eventstore/templates/_helpers.tpl

```diff
@@ -37,3 +37,10 @@ Create DNS used to access the service from within the cluster.
 {{- define "eventstore.dns" -}}
 {{ include "eventstore.fullname" . }}.{{ .Release.Namespace }}.svc.cluster.local
 {{- end -}}
+
+{{- define "eventstore.metrics.image" -}}
```

Comment:
Could we remove this and just use `image` and `imageTag` in the values. See other comment.